src/components/MadeOption.tsx

```diff
@@ -23,6 +23,10 @@ export const MadeOption = (props: Props) => {
     setIsOpenModal(!isOpenModal)
   }, [setIsOpenModal, isOpenModal])
 
+  const closeModal = useCallback(() => {
```

Comment:
How are toggleModal and closeModal different? Is there any case that they are different?

Reply:
> How are toggleModal and closeModal different? Is there any case that they are different?

You are right; In this case it is redundant I have updated it here https://github.com/kixlab/kuiz/pull/46/commits/e1282d380f962fef67e3bab57c6f8bb3d4b5f734.

Reply:
I think CheckDialog should be also updated. For now, there will be (unfunctional) cancel button even if you are not passing closeModal prop.

Reply:
> I think CheckDialog should be also updated. For now, there will be (unfunctional) cancel button even if you are not passing closeModal prop.

That is right; I have updated it here https://github.com/kixlab/kuiz/pull/46/commits/30e81da0c52c18618f82f6fc853513997d88e5b6